utils/vlog/vlog_to_pbtype.py

```diff
@@ -568,5 +872,9 @@ def main(args):
 
 
 if __name__ == "__main__":
+    import doctest
+    failure_count, test_count = doctest.testmod()
+    assert test_count > 0
+    assert failure_count == 0, "Doctests failed!"
     args = parser.parse_args()
```

Comment:
`parser` should not be assigned during import. I think a helper function or defined in `main`. I had gone through my old branches and was going to address this, but seems to make sense as part of this rewrite.

Same applies to `vlog_to_model` which needs a `main`, but that could be here or a different PR.

Reply:
@kgugala